src/main.c

```diff
@@ -174,10 +174,8 @@ main(int argc,
       imlib_create_cropped_scaled_image(0, 0, cwidth, cheight,
                                             twidth, theight);
     if (thumbnail == NULL) {
-      fprintf(stderr, "Unable to create scaled Image: %s\n", strerror(errno));
-      exit(EXIT_FAILURE);
-    }
-    else
+        err(EXIT_FAILURE, "imlib_create_cropped_scaled_image");
```

Comment:
We must avoid informing the user of issues internal to the code since it does not really inform him of the error. Please leave it as it was, but feel free to expand the description of the error if you want.

Reply:
I made changes to your branch resolving conflicts, remember to update your local branch with: `git pull` before making changes.